### PR TITLE
add tmux

### PR DESCRIPTION
**Pull request by harsh18262**
Add `tmux`

### CODE REVIEW

File: packages/tmux/tmux.pacscript [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name="tmux"
+version="3.2a"
+url="https://github.com/tmux/tmux/releases/download/${version}/tmux-${version}.tar.gz"
+build_depends="gcc make pkg-config bison"
+depends="libncurses-dev libevent-dev"
+breaks="tmux-git"
```
Comment (wizard-28) [inline packages/tmux/tmux.pacscript:6]:
```suggestion
breaks="${name}-git ${name}-bin ${name}-deb ${name}-app"
```

Also add the other package types.

File: packages/tmux/tmux.pacscript [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+replace="tmux"
+description="A terminal multiplexer"
+hash="551553a4f82beaa8dadc9256800bcc284d7c000081e47aa6ecbb6ff36eacd05f"
+optdepends=("libutempter-dev: Record user sessions to utmp and wtmp files [add to depends array and rebuild to enable]")
```
Comment (wizard-28) [inline packages/tmux/tmux.pacscript:10]:
```suggestion
optdepends=("libutempter-dev: Record user sessions to utmp and wtmp files")
```

In version `1.6` (currently `develop`) the `optdepends` are called before the installation begins, just like `depends`, so no need to give this warning. The dependency will be available during install.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
https://github.com/archlinux/svntogit-community/blob/packages/tmux/trunk/PKGBUILD

---

**FROM THE AUR**

`install()`:
```
	install -D -m0644 COPYING "$pkgdir/usr/share/licenses/tmux/LICENSE"
```

---

Convert that line to the Pacstall compatible one, and add that.

File: packages/tmux/tmux.pacscript [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+maintainer="Harshwardhan Mehrotra <abrakadabra18262@protonmail.com>"
+prepare() {
+          ./autogen.sh
+}
+build() {
+        ./configure
```
Comment (wizard-28) [inline packages/tmux/tmux.pacscript:16]:
```suggestion
        # Check if `libutempter-dev` ( optdepends ) is installed
        if dpkg-query -W libutempter-dev > /dev/null 2>&1; then
                ./configure --enable-utempter
        else
                ./configure
        fi
```

Suppose the user enables `libutempter`. You need to add the `--enable-utempter` flag here to enable it. My suggestion figures out if the user has installed the lib, and proceeds accordingly.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by harsh18262** [comment]
Suggested Changes added

**Review by wizard-28 — APPROVED** [review]
Well, you could have directly clicked the "Add suggestion to batch"/"Commit suggestion" button, but okay.

File: packages/tmux/tmux.pacscript (hunk 3 above)
Comment (wizard-28) [reply packages/tmux/tmux.pacscript]: Fixed in https://github.com/pacstall/pacstall-programs/pull/361/commits/615b145c09a3c6bd509325f5cba8e977f1e09e50

File: packages/tmux/tmux.pacscript (hunk 2 above)
Comment (wizard-28) [reply packages/tmux/tmux.pacscript]: Fixed in https://github.com/pacstall/pacstall-programs/pull/361/commits/615b145c09a3c6bd509325f5cba8e977f1e09e50

File: packages/tmux/tmux.pacscript (hunk 1 above)
Comment (wizard-28) [reply packages/tmux/tmux.pacscript]: Fixed in https://github.com/pacstall/pacstall-programs/pull/361/commits/615b145c09a3c6bd509325f5cba8e977f1e09e50

**Comment by harsh18262** [comment]
I had to add the command to install the license also so i thought it would create less commits.Will keep in mind for the next time
Thanks

**Pull request merged**